ethereum/rpc/namespaces/eth/api.go

```diff
@@ -870,18 +870,12 @@ func (e *PublicAPI) GetTransactionReceipt(hash common.Hash) (map[string]interfac
 		cumulativeGasUsed += uint64(blockRes.TxsResults[i].GasUsed)
 	}
 
+	// Get the transaction result from the log
 	var status hexutil.Uint
-
-	// NOTE: Response{Deliver/Check}Tx Code from Tendermint and the Ethereum receipt status are switched:
-	// |         | Tendermint | Ethereum |
-	// | ------- | ---------- | -------- |
-	// | Success | 0          | 1        |
-	// | Fail    | 1          | 0        |
```

Comment:
I'd still leave this comment as it is helpful in this context

Reply:
we are not using txResult code anymore so the comment might be irrelevant?

Reply:
we could move it to the docs then

Reply:
okay, done